jobs/views.py

```diff
@@ -120,18 +120,18 @@ def get_prediction(request, pk1, pk2):
                        'prefix_length':1,
                        "rule": "remaining_time",
                        'threshold': 'default',
+                       'log_id':3,
                        }
-    log = Log.objects.get(pk=3)
-    jobrun=JobRun.objects.create(config=config, log=log, type=NEXT_ACTIVITY)
+    jobrun=Job.objects.create(config=config, run=True, type=NEXT_ACTIVITY)
```

Comment:
This method looks like a hack. Update the description to say that this is a demo thing. Or add a TOD